docs/administer/manage_namespaces.md

```diff
@@ -0,0 +1,89 @@
+# Namespaces management
+
+The management of namespaces plays a vital role in efficiently organizing and allocating resources within Percona Everest.
+
+## Add new namespaces
+
+You can run the following command for provisioning a new DB namespace:
+
+```sh
+everestctl namespaces add [NAMESPACE]
+```
+
+The command supports the following flags:
+
+`--operator.mongodb=<bool>`
+
+`--operator.postgresql=<bool>`
+
+`--operator.xtradb-cluster=<bool>`
+
+`--take-ownership` - If the namespace exists, it will be part of Percona Everest. If it is set to false, Percona Everest will throw an error.
+
+??? example "Example"    
+    ```sh
+    everestctl namespaces add production --kubeconfig /Users/rasikachivate/Downloads/Kubeconfig 
+
+    ? Which operators do you want to install? MongoDB    
+    ✓ Installing namespace 'production'
+    ```
+    
+    ```sh
+    everestctl namespaces add development --operator.postgresql=false --kubeconfig /Users/rasikachivate/Downloads/Kubeconfig
+    ✓ Installing namespace 'development'
+    ```
+    In the command above, we did not specify the MongoDB and MySQL  operators. Therefore, by default, Percona Everest assumes these operators are true and will add them.
+
+## Update namespaces
+
+You can run the following command to add more operators to the DB namespace.
+
+
+```sh
+everestctl namespaces update [NAMESPACE] 
+```
+
+!!! note
+    The `everestctl install` command can provision a DB namespace. However, if you run this command again, an error will be thrown: `Everest is already installed.`
```

Comment:
```suggestion
!!! note
    The `everestctl install` command lets you create database namespaces during the initial installation. However, it cannot be rerun to add more database namespaces. To create additional database namespaces after installation, use the namespaces add command.`
```

Reply:
I think we should move this note to the section above, i.e, `Add new namespaces`

Reply:
I agree, this and the 2 bullets below feel out of place